source/funkin/data/character/CharacterRegistry.hx

```diff
@@ -443,13 +443,13 @@ class CharacterRegistry
     }
 
     // If `flipXOffsets` is enabled, we need to account for the default value of `flipX`.
-    if (result.flipX && result.flipXOffsets)
-    {
-      for (anim in result.animations)
-      {
-        anim.offsets[0] *= -1;
-      }
-    }
+    // if (result.flipX && result.flipXOffsets)
+    // {
+    //   for (anim in result.animations)
+    //   {
+    //     anim.offsets[0] *= -1;
```

Comment:
the calculation we actually need is one similar to this:
https://github.com/FunkinCrew/Funkin/blob/fb4bbba3c1160be8fc5ba8b6f6b5aebad69bed72/source/funkin/play/stage/Bopper.hx#L373

Reply:
but im pretty sure even if we have the correct calculation, there will still be edge cases.
so i might just write a python script to automatically set `isFacingRight` to the correct value (just need to hope there is some library that can help me do that)

Reply:
> the calculation we actually need is one similar to this:
> 
> https://github.com/FunkinCrew/Funkin/blob/fb4bbba3c1160be8fc5ba8b6f6b5aebad69bed72/source/funkin/play/stage/Bopper.hx#L373

I see, I should test this change out later.